Resources/Locale/en-US/_Impstation/store/uplink-catalog.ftl

```diff
@@ -1,4 +1,11 @@
 uplink-cleaner-lake-upgrade-name = Shining spring remilitarisation kit
 uplink-cleaner-lake-upgrade-desc = A kit that can be used to remove the safety measures from a shining spring.
+
 uplink-zipper-ap-name = Zipper .22
-uplink-zipper-ap-desc = A regretful purchase from a liquidation sale. Comes loaded with armor-piercing rounds. Uses .22 Auto.
\ No newline at end of file
+uplink-zipper-ap-desc = A regretful purchase from a liquidation sale. Comes loaded with armor-piercing rounds. Uses .22 Auto.
+
+uplink-nemesis-br-name = Nemesis-11 Bundle
+uplink-nemesis-br-desc = Precise and Deadly: the stylish Nemesis-11 Battle Rifle, bundled with 3 magazines.
+
+uplink-rifle-magazine-nemesis-name = Rifle Magazine (.25 Caseless)
+uplink-rifle-magazine-nemesis-desc = Rifle magazine with 24 cartridges. Compatible with the Nemesis-11.
```

Comment:
nitpick: github hates it when there isn't an empty line at the end of files, any files with the "no entry" sign could do with a newline. i realise many of these already existed but they might as well get fixed up while they're easily viewable